### PR TITLE
[zh-cn] Sync 1.25 user-namespaces.md

### PR DESCRIPTION
**Pull request by jzhupup**
Updated text.

```
content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md
```

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+* containerd：计划为 1.7 版本提供支持，
+  更详细信息，请查阅 containerd issue [#7063][containerd-userns-issue]。
+
+<!--
+Support for this in [cri-dockerd is not planned][CRI-dockerd-issue] yet.
+
+[CRI-dockerd-issue]: https://github.com/Mirantis/cri-dockerd/issues/74
+[containerd-userns-issue]: https://github.com/containerd/containerd/issues/7063 
+-->
+在[cri-dockerd 还没有计划][CRI-dockerd-issue] 中提供支持。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:83]: We add a space between English and Mandarin words.

```suggestion
在 [cri-dockerd 还没有计划][CRI-dockerd-issue]中提供支持。
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+{{< glossary_tooltip text="容器运行时" term_id="container-runtime" >}}
+支持。
+
+<!-- 
+* CRI-O: v1.25 has support for user namespaces.
+
+* containerd: support is planned for the 1.7 release. See containerd
+  issue [#7063][containerd-userns-issue] for more details. 
+-->
+* CRI-O：v1.25 已经支持用户名字空间.
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:72]:
```suggestion
* CRI-O：v1.25 已经支持用户名字空间。
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+
+<!-- 
+As a by-product of this, folders and files for these volumes will have
+permissions for the group, even if `defaultMode` or `mode` to specific items of
+the volumes were specified without permissions to groups. For example, it is not
+possible to mount these volumes in a way that its files have permissions only
+for the owner. 
+-->
+作为一个副产品，这些卷的文件夹和文件将具有这个组的权限，
+即使 `defaultMode` 或 `mode` 对卷的特定项目没有指定组的权限。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:315]:
```suggestion
即使 `defaultMode` 或卷中的特定条目对应的 `mode` 没有为所指定的用户组授权也是一样。
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+例如，挂载的这些卷在某种程度上它的的文件不可能只对所有者有权限。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:316]:
```suggestion
例如，不可以设置所挂载的这些卷只允许所有者访问其中的文件。
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+[containerd-userns-issue]: https://github.com/containerd/containerd/issues/7063 
+
+<!-- 
+## Introduction
+
+User namespaces is a Linux feature that allows to map users in the container to
+different users in the host. Furthermore, the capabilities granted to a pod in
+a user namespace are valid only in the namespace and void outside of it. 
+-->
+## 简介
```
[the remark this answers is not in the document]
Comment (jzhupup) [reply content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:95]: 好的

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+doesn't have privileges on the host. You can verify this, for example, if you
+check the user the container process is running `ps` from the host. The user
+`ps` shows is not the same as the user you see if you execute inside the
+container the command `id`. 
+-->
+这意味着容器可以以 一个 root身份运行，并映射到主机上的一个非 root 用户。
+在容器内，进程会认为它是以 root 身份运行（因此像 `apt`、`yum` 等命令可以正常执行），
+而实际上该进程在主机没有这些权限。
+你可以验证这一点，例如，如果你在主机上检查容器进程运行的 `ps` 用户，
+`ps` 用户与你在容器中执行命令 `id` 显示的不同。
```
[the remark this answers is not in the document]
Comment (jzhupup) [reply content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:179]: 好的

**Comment by Sea-n** [comment]
Copy from https://github.com/kubernetes/website/pull/36356#discussion_r957268802:

> @tengqm: 英文原文有问题，需要先修订。
> The English version has some issues that need to be fixed before this PR proceeds.
> /hold

**Comment by tengqm** [comment]
@jzhupup The problem in the English stream has been fixed. Are you interested in pushing this forward?

**Comment by jzhupup** [comment]
> in the English stream has been fixe

ok...looks a bit late

**Pull request closed without merging**